packages/venia-concept/package.json

```diff
@@ -17,9 +17,10 @@
     "build:analyze": "yarn run clean && mkdir dist && yarn run validate-queries && yarn run build:stats",
     "build:dev": "echo 'Skipping venia-concept build...'",
     "build:esm": "BABEL_ENV=development babel src --out-dir esm --root-mode 'upward' --source-maps --copy-files",
-    "build:prod": "webpack -p --color --progress --profile --env.mode production",
+    "build:prod": "webpack -p --color --no-progress --profile --env.mode production",
```

Comment:
Can we just remove the `--no-progress` flag and have the same effect?

Reply:
Yea I don't think this is an option. [Check it](https://webpack.js.org/api/cli/#debug-options).

Reply:
Even if it's not an option, it's good to be explicit about it. (I also think it may be an option, because most CLI parsers for Node apps, like yargs and commander, accept `--no-` prefixes on any boolean flag. This way, the explicit CLI option can override any other setting.

Reply:
`--progress false` would be the explicit way, I think, per the docs.

But that's cool about the `--no-` prefix. The more you no!